src/tests/fuzz/oss-fuzz-configure.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+./configure --disable-shared --enable-fuzztargets --disable-gccmarch-native
```

Comment:
What about adding `sh autogen.sh` and `make` ?

Reply:
Not really seeing the point. The goal here is to place the configure args under our control. For make and autogen we don't need that.